documentation/docs/03-template-syntax/11-bind.md

```diff
@@ -64,16 +77,29 @@ Checkbox and radio inputs can be bound with `bind:checked`:
 </label>
 ```
 
+You can give the input a default value by setting the `defaultChecked` property. This way, when the input is part of a form and its `form.reset()` method is invoked, it will revert to that value instead of `false`. Note that for the initial render the value of the binding takes precedence if it's not `null` or `undefined`.
```

Comment:
```suggestion
If an `<input>` has a `defaultChecked` attribute and is part of a form, it will revert to that value instead of `false` when the form is reset. Note that for the initial render the value of the binding takes precedence unless it is `null` or `undefined`.
```